GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -13,7 +13,9 @@
     </Metadata>
 
     <Installation>
-        <InstallationTarget Version="[14.0,15.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Community" />
```

Comment:
The COM api issue is a breaking change in VS2017 - see here https://developercommunity.visualstudio.com/content/problem/15434/cannot-find-wrapper-assembly-for-type-library-envd.html